daemon/containerd/image.go

```diff
@@ -42,44 +42,10 @@ func (i *ImageService) GetImage(ctx context.Context, refOrID string, options bac
 		platform = platforms.OnlyStrict(*options.Platform)
 	}
 
-	var presentImages []imagespec.DockerOCIImage
-	err = i.walkImageManifests(ctx, desc, func(img *ImageManifest) error {
```

Comment:
I think there's actually a bug in this code; but I found some other code that needs to be looked at as well; see

- https://github.com/moby/moby/pull/47183

Reply:
I added platform filtering. Ptal